tools/qm-change-podman-runtime

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
@slp this script is updating container config in qm.container (which we require) and also podman. Is that good solution to your eyes (only enable when running it)?

Reply:
I still don't think we need to change the runtime globally, but on a per-container basis.